DuckDuckGo/PasswordManager/Bitwarden/Model/BWManager.swift

```diff
@@ -524,7 +541,7 @@ final class BWManager: BWManagement, ObservableObject {
 
     @Published private(set) var status: BWStatus = .disabled {
         didSet {
-            os_log("Status changed: %s", log: .bitwarden, type: .default, String(describing: status))
+            Logger.bitWarden.debug("Status changed: \(String(describing: self.status))")
```

Comment:
isn't default .info?

Reply:
correct, replaced with .log (that uses default)